Libs/RaptureCore/src/main/java/rapture/kernel/BlobApiImpl.java

```diff
@@ -238,10 +238,15 @@ private void lowerStoreBlob(CallingContext context, String blobUri, byte[] conte
     @Override
     public void putBlob(CallingContext context, String blobUri, byte[] content, String contentType) {
         lowerStoreBlob(context, blobUri, content, contentType, false);
-        RaptureURI uri = new RaptureURI(blobUri);
-        BlobRepoConfig repoConfig = Kernel.getRepoCacheManager().getBlobConfig(uri.getAuthority());
-        BlobUpdateObject buo = new BlobUpdateObject(uri, content, contentType);
-        SearchPublisher.publishCreateMessage(context, repoConfig, buo, false);
+        try {
+            RaptureURI uri = new RaptureURI(blobUri, Scheme.BLOB);
+            BlobRepoConfig repoConfig = Kernel.getRepoCacheManager().getBlobConfig(uri.getAuthority());
+            BlobUpdateObject buo = new BlobUpdateObject(uri, content, contentType);
+	    SearchPublisher.publishCreateMessage(context, repoConfig, buo, false);
+        } catch (RaptureException e) {
+            log.error("Unable to index blob" + blobUri + " : " + e.getMessage());
```

Comment:
need a space after "blob" for readability.  why not just log the exception? e.g.

log.error("Unable to index blob " + blobUri, e);